File: examples/different/submissions/accepted/different.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.on('line', (line) => {
    var nums = line.split(' ');
    var a = parseInt(nums[0]);
    var b = parseInt(nums[1]);
    var res = Math.abs(a - b);
    console.log(res);
});

File: examples/different/submissions/accepted/Different.scala
object Different {
    def main(args: Array[String]) {
        var line = scala.io.StdIn.readLine()
        while (line != null) {
            val Array(a, b) = line.split(" ").map(_.toLong)
            val r = scala.math.abs(a - b)
            Console.out.println(r)
            line = scala.io.StdIn.readLine()
        }
    }
}
